### ISSUE TITLE
onAnimatedValueUpdate WARN

### ISSUE THREAD

**Issue by Bardreamaster** [lexiconhq/lexicon#50]
### Describe the bug

Run `/fronted` locally, with prose server deployed on server which is deployed by Docker from `kodefox/prose:latest`. And Test APP on iOS(16.6.1)  latest Expo. And no plugins installed.

There is a warning while click Notifications in Profile page, shown as below(RCTLog.js 36:5). Others looks good.

```
 WARN  Sending `onAnimatedValueUpdate` with no listeners registered.
 WARN  Sending `onAnimatedValueUpdate` with no listeners registered.
 WARN  Sending `onAnimatedValueUpdate` with no listeners registered.
```

### _REQUIRED:_ answer the following questions:
- If this is related to your mobile app:
  - Are you running it through Expo, or did you build it?
  - Is it running on your mobile device or on a simulator?

### To Reproduce
1. Run `api` and `fronted app`
2. Click Notifications in Profile page, or refresh post page, click like button.
4. See warrning.

#### Desktop (please complete the following information):
 - OS: Debian 10

#### Smartphone (please complete the following information):
 - Device:  iPhone
 - OS: iOS 16.6.1

#### GraphQL API (please complete the following information):
Please indicate where the Prose GraphQL API is running.

- On my development machine.

Have you deployed it somewhere? If so, what domain or IP address did you deploy it at?

- prose.robook.org

How specifically did you configure the mobile app to connect to the API?

- Only make fronted app point to ProseUrl.

**Comment by starjustice** [comment]
Hey @Bardreamaster, thank you for opening this issue.

I want to clarify that this warning is not a critical bug that will break the app or render any feature non-functional. However, after conducting our investigation, we suspect that this warning originates from the React Navigation package, as indicated in this issue: https://github.com/react-navigation/react-navigation/issues/7839.

we will try implementing a fix for this warning.

**Comment by Bardreamaster** [comment]
Thanks for help!

**Comment by barrymichaeldoyle** [comment]
Ironically I just ran into this bug in something I was developing today, so very excited to hear a fix is coming 👍  All the workarounds mentioned in https://github.com/react-navigation/react-navigation/issues/7839 make my animations feel less smooth, but my OCD is getting attacked by that warning 😄

**Comment by starjustice** [comment]
I want to inform you that we have already addressed this issue in the [`v2.0.0-beta`](https://github.com/lexiconhq/lexicon/tree/v2.0.0-beta) branch with the following commit: [Commit Link](https://github.com/lexiconhq/lexicon/commit/828ae9ea6e908feab9edce72bb8ec024f12bac19). If you encounter any other issues, please feel free to create a new one.
